roles/libvirt_rhel_vm/tasks/network.yml

```diff
@@ -0,0 +1,18 @@
+- block:
+    - name: create directory to hold domain nic-configs
+      tempfile:
+        state: directory
+      register: _libvirt_tmp_nic_config
+
+    - name: extract destination path
+      set_fact:
+        libvirt_rhel_vm_nic_config_path: "{{ _libvirt_tmp_nic_config.path }}"
+  when: libvirt_rhel_vm_nic_config_path is none
+
+- name: call network-scripts role
+  vars:
+    network_scripts_dest_path: "{{ libvirt_rhel_vm_nic_config_path }}"
+    network_scripts_nics: "{{ libvirt_rhel_vm_domain.nics }}"
+    network_scripts_restart: false
+  import_role:
```

Comment:
I believe it is a bad idea to use `import_role` with `vars`. See https://github.com/pcahyna/meta_standards/blob/initguide/GUIDE.md#setting-variables-on-rolesimport_role-statement ( from https://github.com/oasis-roles/meta_standards/pull/19 )